### PR TITLE
Add feature for viewing submissions and searching by name and form name

### LINKED ISSUE #49: feat:View  and modify status of all the form submissions
## Description

As an Admin, I need to be able to view all the form submissions made and update individual form status.

Changes in the mock: Add a header row, with a checkbox (with all rows selected), Applicant Name, GitHub Username, Zulip Username, Status.

## Mock
https://drive.google.com/file/d/1JuZetC0y8RCQ-72jFrehLSs-8HfTY74A/view

### Update [Required]
- [ ] Admin is able to view submissions
- [ ] Admin is able to add status: accepted/waitlist/rejected/pending to a form 
- [ ] UI should follow the mock with mentioned changes
- [ ] Write tests for it

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1-2 days

### PR DESCRIPTION
**Pull request by bismitaguha**
### Description

- Added list view of submissions
- Added feature for search by form name and user name
- Added feature to view user profile and specific form responses
- In the User profile, you can view their general details, Zulip stats & summary of all forms filled by them

Fixes #49 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

- `submission/`
Screenshot from 2020-08-17 20-19-37

- `submission/:user_id`
Screenshot from 2020-08-28 11-12-39

- `preview/:form_id/:user_id` -> To view the responses of a specific form for a specific user
Screenshot from 2020-08-28 11-12-47

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by bismitaguha** [comment]
CSS work left!

File: src/components/Submission.js
Comment (sidvenu) [inline src/components/Submission.js]:
```suggestion
```

**Review by sidvenu — CHANGES_REQUESTED** [review]
LGTM till now - just remove the test log. Is this PR WIP?

**Comment by sidvenu** [comment]
Is this PR WIP or done? If done, please request a review. If not, please add 'WIP' to the title.

**Review by sidvenu — APPROVED** [review]

**Comment by bismitaguha** [comment]
@abha224 @sidvenu This PR will require changes after the merge of the Zulip Stats PR. Please do not merge before that!!!

**Comment by isabelcosta** [comment]
Will close due to inactivity, and make the issue available again.
Thank you for starting this work @bismitaguha

**Pull request closed without merging**
